lib/sles4sap_publiccloud.pm

```diff
@@ -1019,15 +1019,22 @@ sub wait_for_zypper {
     croak("Argument <instance> missing") unless ($args{instance});
     $args{max_retries} //= 10;
     $args{retry_delay} //= 20;
+    $args{timeout} //= 600;
```

Comment:
eventually change `=over 2` to `=over 4` and  add docstring for the new argument TIMEOUT

Reply:
Done!